cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/parser/YamlParser.java

```diff
@@ -39,7 +41,15 @@ public ParsedSlang parse(SlangSource source) {
             parsedSlang.setName(source.getName());
             return parsedSlang;
         } catch (Throwable e) {
-            throw new RuntimeException("There was a problem parsing the YAML source: " + source.getName() + ".\n" + e.getMessage(), e);
+            String errorMessage = e.getMessage();
+            if (e instanceof ScannerException && errorMessage.startsWith("mapping values")){
+                errorMessage += "Probably did not provide (key: value) pair or missing space after colon(:)";
```

Comment:
Well I was thinking the actual problem was presented first, then a possible solution could be printed. Like in C# there is the error message: The type or namespace name 'a_namespace' could not be found (are you missing a using directive or an assembly reference?).